src/components/logo/logo.tsx

```diff
@@ -1,32 +1,33 @@
-import { HStack, Logo, Link } from "@nypl/design-system-react-components";
+import { Logo, Link } from "@nypl/design-system-react-components";
 import React from "react";
 import appConfig from "appConfig";
 
-const DCLogo = () => {
+const DCLogo = ({ isMobile }) => {
```

Comment:
Create interfaces for components. In this case, it'd be
```
interface DCLogoProps {
  isMobile?: boolean;
}
```

and here `const DCLogo = ({ isMobile = false }: DCLogoProps) => {` to get into the habit of defining prop types for components. I added the "= false" as a default for an example but it doesn't have to be false in this case.